spec_helper.rb

```diff
@@ -36,3 +36,7 @@ def report_on_exception=(value)
   ARGV.unshift $0
   MSpecRun.main
 end
+
+def expect_syntax_error(ruby_src)
```

Comment:
Is this the correct place to put a helper like this? Or should it go in `mspec`?

Reply:
It definitely feels like the wrong place.
Adding `expect_syntax_error` in MSpec feels wrong too, first MSpec never use the term `expect` and it seems far too specific.
You could define it in some fixtures file under language maybe.
But I'm not sure I see the value of this helper, the current code works fine, it's a bit verbose but it's also consistent with all other matches for expected exceptions.

Reply:
IMO duplication in tests is rarely a problem, especially if it's something pretty trivial like this.
It's often even a feature as it makes easier to find out where the error happened and have all the context around vs in N helper methods called around.